src/sysdolphin/baselib/jobj.h

```diff
@@ -84,14 +84,14 @@ typedef struct _HSD_JObj {
     struct _HSD_JObj* next;
     struct _HSD_JObj* parent;
     struct _HSD_JObj* child;
-    u32 flags;
+    u32 x14_flags;
```

Comment:
You don't have to do this with sysdolphin related things. I do need to go back and put comments though about the position.